src/Collapse/withCollapse.tsx

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
 
-import { ItemRecord } from '@graasp/sdk/frontend';
-
 import Collapse from './Collapse';
 
 interface WithCollapseProps {
-  item: ItemRecord;
+  itemName: string;
 }
 
-function withCollapse({ item }: WithCollapseProps) {
+function withCollapse({ itemName }: WithCollapseProps) {
```

Comment:
It might have been easier to use `name` so that the other components could use `withCollapse(item)`, just saying.